corehq/apps/api/odata/serializers.py

```diff
@@ -40,32 +39,33 @@ def to_json(self, data, options=None):
         data['value'] = data.pop('objects')
 
         case_json_list = data['value']
+        case_properties_to_include = get_properties_to_include(domain, case_type)
         for i, case_json in enumerate(case_json_list):
-            self.update_case_json(case_json, domain, case_type)
+            update_case_json(case_json, case_properties_to_include)
 
         return json.dumps(data, cls=DjangoJSONEncoder, sort_keys=True)
 
-    def update_case_json(self, case_json, domain, case_type):
-        properties_to_include = self.get_properties_to_include(domain, case_type)
-        for remove_property in [
-            'id',
-            'indexed_on',
-            'indices',
-            'resource_uri',
-        ]:
-            case_json.pop(remove_property)
-        case_properties = case_json.pop('properties')
-        case_json.update({
-            property_name: case_properties.get(property_name, None)
-            for property_name in properties_to_include
-        })
-
-    @memoized
-    def get_properties_to_include(self, domain, case_type):
-        case_type_to_properties = get_case_type_to_properties(domain)
-        return [
-            'case_name', 'case_type', 'date_opened', 'owner_id', 'backend_id'
-        ] + case_type_to_properties.get(case_type, [])
+
+def get_properties_to_include(domain, case_type):
```

Comment:
it looks like you removed the caching. assume that was intention.

Reply:
yes - I was using caching so the method would only get executed once per request, but now it's only called once without needing caching.